src/trsp/GraphDefinition.cpp

```diff
@@ -404,7 +404,7 @@ int GraphDefinition:: my_dijkstra3(
     GraphEdgeInfo* cur_edge = NULL;
```

Comment:
replace `NULL` with `nullptr` (like 66 of src/transitiveClosure/transitiveClosure_driver.cpp)

Reply:
That line didn't generate a warning in clang, therefore there was no change, this PR is to remove warnings not to improve code near lines that generate warnings.
trsp code I am touching the least as possible